### PR TITLE
Gitlab ci

### PR DESCRIPTION
**Pull request by maktwin**

### CODE REVIEW

File: .gitlab-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (ant31) [inline .gitlab-ci.yaml]: this file should be named with a .yml

Comment (maktwin) [reply .gitlab-ci.yaml]: Ohh🤦 Will fix it :)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  extends: .build-container
+
+
+# TAGS IMAGES
+printer tag gitsha:
+  extends: .tag-container
+  variables:
+    DEST_TAG: $CI_COMMIT_SHORT_SHA
+  rules:
+    - if: '$CI_COMMIT_BRANCH == "gitlab_ci"'
```
Comment (tiarly) [inline .gitlab-ci.yml:31]: I believe this was intentional for testing purposes, but for merging it would `master`  right?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    - if: '$CI_COMMIT_TAG'
+
+printer tag:
+  extends: .tag-container
+  rules:
+    - if: '$CI_COMMIT_BRANCH == "gitlab_ci"'
```
Comment (tiarly) [inline .gitlab-ci.yml:37]: same as above

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    - if: '$CI_COMMIT_BRANCH == "production"'
+    - if: '$CI_COMMIT_TAG'
+  variables:
+    DEST_TAG: $CI_COMMIT_REF_SLUG
+
+printer tag latest:
+  extends: .tag-container
+  rules:
+    - if: '$CI_COMMIT_BRANCH == "gitlab_ci"'
```
Comment (tiarly) [inline .gitlab-ci.yml:46]: same

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+# Deploy
+deploy-staging:
+  needs:
+    - printer tag gitsha
+  extends: .deploy
+  stage: deploy
+  variables:
+    DEPLOY_ENV: conny-staging
+  rules:
+    - if: '$CI_COMMIT_BRANCH == "gitlab_ci"'
```
Comment (tiarly) [inline .gitlab-ci.yml:61]: and here

File: Dockerfile.p [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+COPY package.json ./
+COPY yarn.lock ./
+
+RUN yarn install
+
+COPY . .
+COPY ./fonts/ /usr/share/fonts
+
+# RUN yarn build
```
Comment (ant31) [inline Dockerfile.p:35]: why is it not needed anymore?

**Comment by ant31** [comment]
Commit cherry-picked on the other PR

**Pull request closed without merging**